ext/standard/http_fopen_wrapper.c

```diff
@@ -726,113 +726,113 @@ static php_stream *php_stream_url_wrap_http_ex(php_stream_wrapper *wrapper,
 			}
 			ZVAL_STRINGL(&http_response, tmp_line, tmp_line_len);
 			zend_hash_next_index_insert(Z_ARRVAL_P(response_header), &http_response);
-		}
```

Comment:
This change looks more complicated than it needs to be, I think. Wouldn't it be sufficient to add an `else` here that reports an error, similar to what is done just below?

Reply:
It's sufficient but we add new code. I just updated the implementation.